### PR TITLE
Fix CodeQL3000 tsaoptions.json

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

File: tsaoptions.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "areaPath": "DevDiv\\ASP.NET Core",
```
Comment (dougbu) [inline tsaoptions.json:2]: @Tratcher @danmoseley I bet this isn't the correct area path for Katana. What's right❔

**Review by dougbu — APPROVED** [review]
Looks good once you have the right area path

File: tsaoptions.json (hunk 1 above)
Comment (Tratcher) [reply tsaoptions.json:2]: 🤷

Comment (mkArtakMSFT) [reply tsaoptions.json:2]: DevDiv\ASP.NET OOB\Katana

**Pull request merged**